cmd/traffic_cop/traffic_cop.cc

```diff
@@ -657,9 +656,6 @@ config_reload_records()
   config_read_int("proxy.config.cop.source_port", &source_port, true);
 #endif
 
-  config_read_int("proxy.local.cluster.type", &tmp_int);
-  cluster_type = static_cast<MgmtClusterType>(tmp_int);
-
```

Comment:
There is another patch that I have as a WIP that is for totally removing it. It touches more things than it first appears.